index.bs

```diff
@@ -187,6 +187,10 @@ After several weeks, she has developed a tiny DNN model that can even run on
 CPU. In order to accommodate CPU execution, she modifies the application
 so that the application loads the tiny model in the case of CPU-only devices.
 
+### Operation Level Execution ### {#usecase-op-level-exec}
+
+A JavaScript ML framework is responsible for loading, interpreting and executing a ML model. During the model execution phase, the framework iterates through the operations of the model and executes each operation on the hardware device, like CPU or GPU. To avoid the unnecessary data copying cross devices, the framework selects the same device to execute the operations. For a compute intensive operation, such as convolution 2D or matrix multiplication, the framework uses WebNN API to execute it with the ML-specific acceleration available on that selected device.
```

Comment:
s/cross devices/across devices/

Reply:
Thanks! Fixed it in the new commit https://github.com/webmachinelearning/webnn/pull/154/commits/2af623828791bf6ea310fd3b5467e10e687e84b7